brayns/engine/Model.cpp

```diff
@@ -37,6 +37,12 @@ const std::map<std::string, std::string> &Model::getMetaData() const noexcept
     return _metadata;
 }
 
+void Model::onInspect(const Vector3f &hit, OSPGeometricModel modelHandle, uint32_t primitiveID, JsonObject &writeResult)
```

Comment:
Just some questions/remarks:

- I would put the 3 first arguments in a struct for flexibility.
- modelHandle is always the model the component is attached to right ? So we could access it without passing it in argument no ?
- Is there any reason why the result is an output parameter and not a return value ?